sigs.yaml

```diff
@@ -2732,6 +2732,10 @@ sigs:
       github: aojea
       name: Antonio Ojea
   subprojects:
+  - name: inference-perf
```

Comment:
nit: same here - move to the bottom

Reply:
https://github.com/kubernetes/community/pull/8247#discussion_r1921117672